web/src/main/java/org/fao/geonet/monitor/onlineresource/wps/CallableWpsCheck.java

```diff
@@ -97,7 +97,7 @@ private String getRequestXml() {
                 "    <wps:Input>\n" +
                 "      <ows:Identifier>subset</ows:Identifier>\n" +
                 "      <wps:Data>\n" +
-                "        <wps:LiteralData>LATITUDE,-31.6855,-31.6855;LONGITUDE,114.8291,114.8291</wps:LiteralData>\n" +
+                "        <wps:LiteralData>LATITUDE,-31.6855,-31.6855;LONGITUDE,114.8291,114.8291;DEPTH,0,100</wps:LiteralData>\n" +
```

Comment:
But this only applies to CARS.  Would it be better to make a DEPTH subset parameter optional?